### ISSUE TITLE
Input field of component is not selectable or is disabled

### ISSUE THREAD

**Issue by tonylisanti** [DanielPantle/strapi-plugin-react-icons#1]
Running strapi v4.10.4. Had the same issue with v4.9.2 as well.

I've installed the plugin via npm which installs v0.1.3 and then I imported all the fonts as described in the documentation but the input field for the added component is not selectable or is disabled:

[screencast-localhost_1337-2023.05.17-14_09_52.webm](https://github.com/DanielPantle/strapi-plugin-react-icons/assets/232865/53a0f367-88ca-4fd0-9d23-2d65311ac8f9)

Thanks.

**Comment by tonylisanti** [comment]
@prestonator that was the case. Might should be a button instead of an input field.

**Comment by kienbui195** [comment]
> Running strapi v4.10.4. Had the same issue with v4.9.2 as well.
> 
> I've installed the plugin via npm which installs v0.1.3 and then I imported all the fonts as described in the documentation but the input field for the added component is not selectable or is disabled:
> 
>  screencast-localhost_1337-2023.05.17-14_09_52.webm 
> Thanks.

try to change to light mode and enjoy

**Comment by kienbui195** [comment]
![image](https://github.com/DanielPantle/strapi-plugin-react-icons/assets/106719805/d1871ddd-5c6f-4eca-8d03-31c18c1a7fee)
I think in dark mode the pile of icons looks a bit alum

**Comment by tonylisanti** [comment]
@kienbui195 Ahh... yes, the search/magnify icon shows in light mode.

**Comment by prestonator** [comment]
@tonylisanti @kienbui195 I don't have much experience with contributing, so I just forked, added a couple features I wanted and published to npm under the name [strapi-react-icons](https://www.npmjs.com/package/strapi-react-icons). 

I added a search feature, it also displays all the icons at the same time. Feel free to take it and do whatever you want, if you do use it, I would fork, because I'll probably be working on it a little here and there and don't want to mess with anyone's projects.
